docs/environment-and-software/user-environment/package-managers/uv.md

```diff
@@ -142,30 +124,98 @@ uv run python analyze_data.py
-anything.
+The `uv run` command automatically activates your project environment and keeps it in sync with your `pyproject.toml` and lockfile, guaranteeing your command runs in a consistent, locked environment.
+
+You can use `uv pip list` to see installed packages in your project environment:
+
+```bash
+uv pip list
+```
 
 !!! note "Project environments"
     `uv` stores project environments in a `.venv` directory within your
     project. You do not need to activate this environment manually when
     using `uv run`. Alternatively, you can activate it traditionally with
     `source .venv/bin/activate`; then use `python` as usual.
 
+---
 
-## Reproducing `uv` environments
+## Creating and Activating a Virtual Environment with `uv`
 
-One of `uv`'s key strengths is built-in support for reproducibility
-through lock files.
+`uv` can be used to create standalone virtual environments similar to `python -m venv`.
 
-### Using lock files
+To create an environment:
 
-When you create a `uv` project and add dependencies, `uv` automatically
-generates a `uv.lock` file that specifies exact versions of all
-packages and their dependencies. 
+```bash
+mkdir -p /glade/work/$USER/uv-envs
+
+# Create a new uv virtual environment
+uv venv /glade/work/$USER/uv-envs/myenv
+
+# Activate the environment
+source /glade/work/$USER/uv-envs/myenv/bin/activate   
+```
+Once activated, you can use `python` commands within this environment just like a conda environment. If you use `uv run` commands, you do not need to activate the environment manually.
+
+!!! info "Specifying Python Versions with `uv`"
+    You can create virtual environments with specific Python versions using the `--python` flag.
+
+    ```bash
+    uv venv /glade/work/$USER/uv-envs/myenv-py3.10 --python python3.10
+
+    source /glade/work/$USER/uv-envs/myenv-py3.10/bin/activate
+
+    python --version
+    ```
+    If you do not specify a Python version, `uv` will use the default Python version available in your environment.
+
+!!! note "Environment location"
+    Create virtual environments in `/glade/work/$USER` to avoid filling
+    your home directory quota. Virtual environments can be several MBs-GBs 
+    or larger depending on installed packages.
 
-To reproduce an environment from a project with a lock file:
 
+### Low-level `uv pip` Commands
+These commands are intended for legacy workflows or cases where the high-level project commands (`uv init`, `uv add`, `uv run`) do not provide enough control.
+
+Once your `uv` environment is active, you can install packages using uv’s pip interface:
+```bash
+uv pip install numpy pandas matplotlib
+```
+
+This behaves similarly to `pip`, but with **much** faster dependency resolution.
+
+Similarly you can install from a `requirements.txt` file:
+```bash
+uv pip install -r requirements.txt
+```
+
+or you can uninstall packages:
+```bash
+uv pip uninstall numpy
+```
+
+Other useful `uv pip` commands include:
```

Comment:
These don't seem to be rendering as bullets on separate lines but instead as a paragraph.